tests/bluetooth/df/connectionless_cte_chains/src/common.c

```diff
@@ -141,8 +141,8 @@ void common_create_per_adv_chain(struct ll_adv_set *adv_set, uint8_t pdu_count)
 	pdu = lll_adv_sync_data_peek(lll_sync, NULL);
 	ull_adv_sync_pdu_init(pdu, 0);
 
-	err = ull_adv_sync_pdu_alloc(adv_set, 0, 0, NULL, &pdu_prev, &pdu, &extra_data_prev,
-				     &extra_data, &pdu_idx);
+	err = ull_adv_sync_pdu_alloc(adv_set, ULL_ADV_PDU_EXTRA_DATA_ALLOC_IF_EXIST, &pdu_prev,
+				     &pdu, &extra_data_prev, &extra_data, &pdu_idx);
```

Comment:
Why is the function use (9 vs 7 parameters) in this test file changed as part of this PR? Are these tests not built/run in CI?

Reply:
This change is done in separate commit. There was a compilation error related with wrong parameters list. Now if fits to the function declaration
```
uint8_t ull_adv_sync_pdu_alloc(struct ll_adv_set *adv,
			       enum ull_adv_pdu_extra_data_flag extra_data_flags,
			       struct pdu_adv **ter_pdu_prev, struct pdu_adv **ter_pdu_new,
			       void **extra_data_prev, void **extra_data_new, uint8_t *ter_idx);
```